amplfi/train/models/flow.py

```diff
@@ -164,55 +197,73 @@ def test_step(self, batch, _) -> AmplfiResult:
             descaled, mask = self.filter_parameters(descaled)
             log_probs = log_probs[mask]
 
-        parameters = self.scale(parameters, reverse=True)
-        parameters = parameters.cpu().numpy()[0]
+        # convert samples to dictionary for
+        # calculating log probabilites
+        samples_dict = dict(
+            zip(
+                self.hparams.inference_params,
+                descaled.transpose(1, 0),
+                strict=True,
+            )
+        )
 
-        # create a dictionary of injection parameters
-        # mapping from parameter string to the true injection value
-        # and add snr if provided
-        injection_parameters = {
-            k: float(v)
-            for k, v in zip(self.inference_params, parameters, strict=False)
-        }
-        injection_parameters["ra"] = injection_parameters["phi"]
+        log_priors = self.training_prior.log_prob(samples_dict)
```

Comment:
I initially though, this was just log of the prior. Can we use a different variable name, like `log_prob_on_training_prior`? Also add a comment, that this is the posterior samples density estimate on the prior distribution.

Reply:
To clarify, this is the prior probability of the sampled posterior samples using the training prior. 

Changed the variable name to `log_prior_of_posterior_samples`